app/controllers/games_controller.rb

```diff
@@ -1,5 +1,18 @@
 class GamesController < ApplicationController 
 	def show 
 		render template: "games/#{params[:game]}"
-	end
+	end 
+
+	def new 
+		@game = Game.new 
+	end 
+
+	def create 
+		@game = Game.create 
+	end 
+
+	def update 
+		@game = Game.find(params[:id]) 
+		current_player.update_attribute(:game_id, @game.id) 
```

Comment:
Hmm, @wdehaes , you probably are right, but in order to join a game, you need to have a player, in which case we will have a current player... We don't have it right now, but we will in the future, so should I leave it like that or try the thing with the object you mentioned?